python/osrd_schemas/osrd_schemas/rolling_stock.py

```diff
@@ -196,36 +203,46 @@ class RollingStock(BaseModel, extra=Extra.forbid):
 
     railjson_version: RAILJSON_ROLLING_STOCK_VERSION_TYPE = Field(default=RAILJSON_ROLLING_STOCK_VERSION)
     version: int = Field(default=0, description="Rolling stock version")
-    name: constr(max_length=255)
+    name: str = Field(max_length=255)
     locked: bool = Field(default=False, description="Whether the rolling stock can be edited/deleted or not")
     effort_curves: EffortCurves = Field(description="Curves mapping speed (in m/s) to maximum traction (in newtons)")
     base_power_class: Optional[str] = Field(
-        description="The power usage class of the train (optional because it is specific to SNCF)"
+        description="The power usage class of the train (optional because it is specific to SNCF)", default=None
     )
     power_restrictions: Optional[PowerRestrictions] = Field(
-        description="Mapping from train's power restriction codes to power classes"
+        description="Mapping from train's power restriction codes to power classes", default=None
     )
     length: PositiveFloat = Field(description="The length of the train, in m")
     max_speed: PositiveFloat = Field(description="Maximum speed in m/s")
-    startup_time: confloat(ge=0) = Field(description="The time the train takes before it can start accelerating in s")
-    startup_acceleration: confloat(ge=0) = Field(description="The maximum acceleration during startup in m/s^2")
+    startup_time: Annotated[float, Field(ge=0)] = Field(
+        description="The time the train takes before it can start accelerating in s"
+    )
+    startup_acceleration: Annotated[float, Field(ge=0)] = Field(
+        description="The maximum acceleration during startup in m/s^2"
+    )
```

Comment:
```suggestion
    startup_time: float = Field(ge=0, description="The time the train takes before it can start accelerating in s")
    startup_acceleration: float = Field(ge=0, description="The maximum acceleration during startup in m/s^2")
```